src/main/java/edu/cornell/kfs/fp/batch/xml/AccountingXmlDocumentListener.java

```diff
@@ -0,0 +1,50 @@
+package edu.cornell.kfs.fp.batch.xml;
+
+import javax.xml.bind.Unmarshaller;
+import javax.xml.bind.ValidationEvent;
+import javax.xml.bind.ValidationEventHandler;
+
+public class AccountingXmlDocumentListener extends Unmarshaller.Listener implements ValidationEventHandler {
```

Comment:
Should the class name be AccountingXmlDocumentUnmarshalListener since this for unmarshaling?

Reply:
Good idea, I'll rename this class accordingly.